translations/catroweb.en.yml

```diff
@@ -848,6 +853,7 @@ catro-notifications:
   follow:
     title: 'New Follower!'
     message: 'User %user_link% follows you now.'
+    message2: '%user_link%'
```

Comment:
why would you need something like this in the translations?
No one will ever be able to translate it